DPG/Tasks/TPC/tpcSkimsTableCreator.cxx

```diff
@@ -0,0 +1,532 @@
+  /// Configurables kaon
+  Configurable<float> invariantMassCutK0Short{"invariantMassCutK0Short", 0.5, "Mass cut for K0short"};
+  Configurable<float> cutQTK0min{"cutQTK0min", 0.1075, "Minimum qt for K0short"};
+  Configurable<float> cutQTK0max{"cutQTK0max", 0.215, "Minimum qt for K0short"};
+  Configurable<float> cutAPK0pinner{"cutAPK0pinner", 0.199, "inner momentum for K0short"};
+  Configurable<float> cutAPK0pouter{"cutAPK0pouter", 0.215, "outer momentum for K0short"};
+  Configurable<float> cutAPK0alinner{"cutAPK0alinner", 0.8, "inner alpha for K0short"};
+  Configurable<float> cutAPK0alouter{"cutAPK0alouter", 0.9, "outer alpha for K0short"};
+  /// Configurables lambda/antilambda
+  Configurable<float> invariantMassCutLambda{"invariantMassCutLambda", 0.5, "Mass cut for Lambda and Anti-Lambda"};
+  Configurable<float> cutQTmin{"cutQTmin", 0.03, "the minimum value of qt for Lambda"};
+  Configurable<float> cutAlphaminL{"cutAlphaminL", 0.35, "minimum alpha for Lambda"};
+  Configurable<float> cutAlphamaxL{"cutAlphamaxL", 0.7, "maximum alpha for Lambda"};
+  Configurable<float> cutAlphaminAL{"cutAlphaminAL", -0.7, "minimum alpha for Anti-Lambda"};
+  Configurable<float> cutAlphamaxAL{"cutAlphamaxAL", -0.35, "maximum alpha for Anti-Lambda"};
+  Configurable<float> cutAPL0{"cutAPL0", 0.10, "cutAPL par0"};
+  Configurable<float> cutAPL1{"cutAPL1", 0.69, "cutAPL par1"};
+  Configurable<float> cutAPL2{"cutAPL2", 0.5, "cutAPL par2"};
+  /// Configurables gamma
+  Configurable<float> gammaAsymmetryMax{"gammaAsymmetryMax", 0.95, "maximum photon asymetry."};
+  Configurable<float> gammaPsiPairMax{"gammaPsiPairMax", 0.1, "maximum psi angle of the track pair. "};
+  Configurable<float> gammaQtPtMultiplicator{"gammaQtPtMultiplicator", 0.11, "Multiply pt of V0s by this value to get the 2nd denominator in the armenteros cut. The products maximum value is fV0QtMax."};
+  Configurable<float> gammaQtMax{"gammaQtMax", 0.040, "the maximum value of the product, that is the maximum qt"};
+  Configurable<float> gammaV0RadiusMin{"gammaV0RadiusMin", 5., "the minimum V0 radius of the gamma decay"};
+  Configurable<float> gammaV0RadiusMax{"gammaV0RadiusMax", 180., "the maximum V0 radius of the gamma decay"};
+  Configurable<float> cutAlphaG1{"cutAlphaG1", 0.35, "maximum alpha gamma decay cut 1"};
+  Configurable<float> cutAlphaG2min{"cutAlphaG2min", 0.6, "minimum alpha gamma decay cut 2"};
+  Configurable<float> cutAlphaG2max{"cutAlphaG2max", 0.8, "maximum alpha gamma decay cut 2"};
+  Configurable<float> cutQTG{"cutQTG", 0.04, "maximum qt gamma decay"};
+
+  /// Kaon selection
+  template <typename C, typename V0>
+  bool selectionKaon(C const& collision, V0 const& v0)
+  {
+    // initialise dynamic variables
+    float alpha = v0.alpha();
+    float qt = v0.qtarm();
+    float cosPA = v0.v0cosPA(collision.posX(), collision.posY(), collision.posZ());
+    float q_K = {cutAPK0pinner * sqrt(abs(1 - ((alpha * alpha) / (cutAPK0alinner * cutAPK0alinner))))};
```

Comment:
```suggestion
    const float alpha = v0.alpha();
    const float qt = v0.qtarm();
    const float cosPA = v0.v0cosPA(collision.posX(), collision.posY(), collision.posZ());
    const float q_K = {cutAPK0pinner * sqrt(abs(1 - ((alpha * alpha) / (cutAPK0alinner * cutAPK0alinner))))};
```
please also check below which other variables can be declared `const`. I think it is for most of the variables...